1.x-src/NodaTime.Tzdb/PackagedTzdb.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2016 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NodaTime;
+using NodaTime.TimeZones;
+using System.IO;
+using System.Reflection;
+
+namespace NodaTime.Tzdb
+{
+    /// <summary>
+    /// Holder class for the TZDB data in the <c>NodaTime.Tzdb</c>
+    /// package, which is updated when IANA ships a new data release.
+    /// </summary>
+    public class PackagedTzdb
```

Comment:
Should definitely be static - will fix that now.

Tzdb would be fine if it weren't in a namespace called Tzdb. We could put it in NodaTime.TimeZones instead, although it would be slightly unconventional to have a package name of NodaTime.Tzdb without that as a namespace anywhere.

Definitely up for more brainstorming on this.